zh_tw.lang

```diff
@@ -0,0 +1,209 @@
+gui.xaero_medium =中型
+gui.xaero_large =大型
+gui.xaero_pvp_sprint = PVP衝鋒
+gui.xaero_quick_eat =快速進食
+gui.xaero_toggle_sneak =切換潛行
+gui.xaero_quick_use_1 =快速使用1
+gui.xaero_quick_use_2 =快速使用2
+gui.xaero_quick_use_3 =快速使用3
+gui.xaero_quick_use_4​​ =快速使用4
+gui.xaero_quick_use_5 =快速使用5
+gui.xaero_quick_use_6 =快速使用6
+gui.xaero_quick_use_7 =快速使用7
+gui.xaero_quick_use_8 =快速使用8
+gui.xaero_quick_use_9 =快速使用9
+gui.xaero_zoom_in =小地圖縮小
+gui.xaero_zoom_out =小地圖放大
+gui.xaero_on =開
+gui.xaero_off =關
+gui.xaero_restores =恢復值
+gui.xaero_preset_topleft =左上
+gui.xaero_preset_topright =右上
+gui.xaero_waypoint_teleport =傳送
+gui.xaero_waypoints_key =路徑點
+gui.xaero_display_coords =顯示坐標
+gui.xaero_disable_enable =關閉/啟用
+gui.xaero_disabled =關閉
+gui.xaero_minimap_guide =點擊大規模移動小地圖
+gui.xaero_change_position =改變位置
+gui.xaero_preset_bottom_left =左下
+gui.xaero_preset_bottom_right =右下
+gui.xaero_ingame_waypoints =遊戲內顯示路徑點
+gui.xaero_next =下一頁>>
+gui.xaero_previous = <<上一頁
+gui.xaero_lock_north =鎖定南北
+gui.xaero_deathpoint =最後一次死亡地點
+gui.xaero_deathpoint_old =之前的死亡地點
+gui.xaero_deathpoints =死亡地點
+gui.xaero_chunkgrid =區塊鏈
+gui.xaero_entity_colours =實體顏色
+gui.xaero_players =玩家
+gui.xaero_mobs =友好的生物
+gui.xaero_hostile =敵對的生物
+gui.xaero_items =物品
+gui.xaero_other =其他實體
+gui.xaero_hunger_low =新生兒值過低
+gui.xaero_hp_low =血量過低
+gui.xaero_enlarge_map =擴大小地圖
+gui.xaero_fbo =測試版
+gui.xaero_potion_status_settings =藥水狀態設置
+gui.xaero_potion_names =藥水效果名稱
+gui.xaero_opacity =不更長
+gui.xaero_slime_chunks =史萊姆區塊鏈
+gui.xaero_unlimited =無限制
+gui.xaero_old_deathpoints =保持舊的死亡點
+gui.xaero_toggle_map =開關小地圖
+gui.xaero_antialiasing =抗鋸齒
+gui.xaero_safe_mode =安全模式
+gui.xaero_transfer =移動
+gui.xaero_transfer_all =移動所有標記
+gui.xaero_clear =清除坐標
+gui.xaero_from =移動
+gui.xaero_to =到
+gui.xaero_used_seed =種子使用：
+gui.xaero_show_distance =距離標記
+gui.xaero_waypoints_scale =標記大小
+gui.xaero_block_colours =方塊顏色
+gui.xaero_accurate =準確
+gui.xaero_vanilla =普通
+gui.xaero_lighting =燈光
+gui.xaero_air_setting =低氧提醒
+gui.xaero_air_low =低氧
+gui.xaero_looking_at =看到
+gui.xaero_all =全部
+gui.xaero_display_redstone =顯示紅石
+gui.xaero_clear_set_message =你確定你想清除當前設置嗎
+gui.xaero_clear_set_message2 =所有路徑點重置設置中刪除。
+gui.xaero_default =默認
+gui.xaero_create_set =新的路徑點設置
+gui.xaero_world =世界
+gui.xaero_waypoint_set =設置
+gui.xaero_delete_set =刪除設置
+gui.xaero_delete_set_message =你確定你想刪除當前設置嗎
+gui.xaero_delete_set_message2 =此設置重置當前世界中可移除
+gui.xaero_toggle_waypoints =切換路徑點
+gui.xaero_dots_scale =實體顯示範圍
+gui.xaero_entity_info =實體信息
+gui.xaero_toggle_slime =切換史萊姆區塊鏈
+gui.xaero_toggle_grid =切換區塊網格
+gui.xaero_compass_over_wp =指南針上的路徑點
+gui.xaero_display_hostile =顯示敵對的生物
+gui.xaero_box_entity_info =顯示您所指向的生物實體信息界面
+gui.xaero_entity_info_settings =實體信息設置
+gui.xaero_entity_info_stay =顯示實體信息
+gui.xaero_entity_info_distance =可達距離
+gui.xaero_entity_info_max_hearts =最大生命值
+gui.xaero_current_biome =顯示生物群系
+gui.xaero_auto_map_size =自動
+gui.xaero_reset_message =你確定你想重新設置設置嗎？
+gui.xaero_reset_message2 =此操作無法恢復。
+gui.xaero_entity_info_numbers =生命值
```

Comment:
Does this mean displaying numbers instead of hearts?

Reply:
Yes, the word "值" is value, so this should be numbers. But we called hearts in Minecraft as "生命值" too. So it's a bit confusing. To avoid that we can change it to "生命值（數值）".